pepper-apis/src/test/java/org/broadinstitute/ddp/model/event/NotificationEventActionTest.java

```diff
@@ -116,20 +114,15 @@ public void test_invitationEmail_providesInvitationIdSubstitution() {
     public void test_invitationEmail_usesLatestNonVoidedInvitation() {
         TransactionWrapper.useTxn(handle -> {
             var invitationFactory = handle.attach(InvitationFactory.class);
-            InvitationDto invitation1 = invitationFactory.createInvitation(InvitationType.AGE_UP,
+            InvitationDto invitation1 = invitationFactory.createAgeUpInvitation(
                     testData.getStudyId(), testData.getUserId(), "test111@datadonationplatform.org");
-            InvitationDto invitation2 = invitationFactory.createInvitation(InvitationType.AGE_UP,
+            InvitationDto invitation2 = invitationFactory.createAgeUpInvitation(
                     testData.getStudyId(), testData.getUserId(), "test222@datadonationplatform.org");
-            try {
-                Thread.sleep(2);
-            } catch (InterruptedException e) {
-                fail("Got interrupted while sleeping");
-            }
```

Comment:
@MocanaAtBroad I kind of think that we got into that timing issue on CircleCI because we were only using millisecond precision. I have expanded precision of these timestamps, so hopefully we don't run into that timing issue again and give us 2 seconds back :)